pages/applications/add-case.tsx

```diff
@@ -55,12 +57,22 @@ export default function AddCasePage({ user }: PageProps): JSX.Element {
       assignedTo: user.email,
     };
 
-    createApplication(request).then(() => {
-      router.push({
-        pathname: '/applications/view-register',
-        query: { status: ApplicationStatus.MANUAL_DRAFT },
+    createManualApplication(request);
+  };
+
+  const createManualApplication = async (request: Application) => {
+    try {
+      const newApplication = await createApplication(request);
+      const completedApplication = await completeApplication(newApplication);
+      const setToManualDraft = await updateApplication({
+        ...completedApplication,
+        status: ApplicationStatus.MANUAL_DRAFT,
```

Comment:
This definitely feels like one of those situations where @anefiox could help you out to streamline this flow. But I guess this would work too.

Reply:
Agreed. Maybe a future ticket so we don't have to call 3 different endpoints.